nipy/algorithms/graph/tests/test_forest.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
+
```

Comment:
Yes, but that works in any Python version, and is printing a tuple:

```
$ python2.6 -c 'print("some", "strings")'
('some', 'strings')
$ python2.7 -c 'print("some", "strings")'
('some', 'strings')
$ python3.4 -c 'print("some", "strings")'
some strings
```

But maybe you meant - could I just print the tuple so it looks like it's a print function call?